docs/collections/electric-collection.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Electric Collection
+---
+
+# Electric Collection
+
+Electric collections provide seamless integration between TanStack DB and ElectricSQL, enabling real-time data synchronization with your Postgres database through Electric's sync engine.
+
+## Overview
+
+The `@tanstack/electric-db-collection` package allows you to create collections that:
+- Automatically sync data from Postgres via Electric shapes
+- Support optimistic updates with transaction confirmation and automatic rollback on errors
+- Handle persistence through customizable mutation handlers
+
+## Installation
+
+```bash
+npm install @tanstack/electric-db-collection @tanstack/db
+```
+
+## Basic Usage
+
+```typescript
+import { createCollection } from '@tanstack/db'	
+import { electricCollectionOptions } from '@tanstack/electric-db-collection'
+
+const todosCollection = createCollection(
+  electricCollectionOptions({
+    shapeOptions: {
+      url: 'https://example.com/v1/shape',
+      params: {
+        table: 'todos',
+      },
+    },
+    getKey: (item) => item.id,
+  })
+)
+```
+
+## Configuration Options
+
+The `electricCollectionOptions` function accepts the following options:
+
+### Required Options
+
+- `shapeOptions`: Configuration for the ElectricSQL ShapeStream
+- `getKey`: Function to extract the unique key from an item
+
+### Shape Options
+
+- `url`: The URL to your Electric sync service
+- `params`: Shape parameters including:
```

Comment:
Same, let's not document this as the normal pattern. Show adding these in the proxy.

Reply:
I've added the proxy example and mention the other shape parameters there